test/utils/manifests.go

```diff
@@ -86,12 +86,42 @@ func TestingMachineProviderSpec(awsCredentialsSecretName string, clusterID strin
 		},
 		PublicIP: &publicIP,
 	}
+}
+
+func TestingMachineProviderSpec(awsCredentialsSecretName string, clusterID string) (clusterv1alpha1.ProviderSpec, error) {
+	machineProviderSpec := testingAWSMachineProviderSpec(awsCredentialsSecretName, clusterID)
+	codec, err := providerconfigv1.NewCodec()
+	if err != nil {
+		return clusterv1alpha1.ProviderSpec{}, fmt.Errorf("failed creating codec: %v", err)
+	}
+	config, err := codec.EncodeProviderSpec(machineProviderSpec)
+	if err != nil {
+		return clusterv1alpha1.ProviderSpec{}, fmt.Errorf("EncodeToProviderConfig failed: %v", err)
+	}
+	return *config, nil
+}
 
+func TestingMachineProviderSpecWithEBS(awsCredentialsSecretName string, clusterID string) (clusterv1alpha1.ProviderSpec, error) {
+	rootDeviceName := "/dev/sdb1"
```

Comment:
why do you set "/dev/sdb1" and then the expectation is https://github.com/openshift/cluster-api-provider-aws/pull/137/files#r248330908

Reply:
Removed all of that, it's now up to the AMI.